src/eve-server/market/MarketDB.cpp

```diff
@@ -358,17 +357,47 @@ PyRep *MarketDB::GetTransactions(uint32 clientID, Market::TxData& data) {
         buy += std::to_string(data.isBuy);
     }
 
+    std::string client = "";
+    if (data.clientID) {
+        client = "AND clientID=";
+        client += std::to_string(data.clientID);
+    }
+
```

Comment:
Allows client to be queried, if desired. But currently this is effectively unused, just like before (technically).